### DISCUSSION TITLE
how to read log files of mobile using flet.

### DISCUSSION THREAD

**Discussion by saisaigraph** [flet-dev/flet#1097] [General]
Any help or pointers how to read log files of mobile using flet.
Thanks
Sai

**Comment by bobwatcherx** [comment]
Reading log files of mobile devices is a useful technique for debugging and troubleshooting issues on mobile devices. Here are some general steps to help you get started with reading log files of mobile devices:

    Connect your mobile device to your computer: Use a USB cable to connect your mobile device to your computer. Make sure that USB debugging is enabled on your mobile device.

    Install ADB: ADB (Android Debug Bridge) is a command-line tool that lets you communicate with your Android device. You can install ADB on your computer by downloading the Android SDK platform-tools from the official Android website.

    Enable USB debugging: On your mobile device, go to Settings > Developer options > USB debugging, and enable USB debugging.

    Use ADB to extract log files: Open a terminal or command prompt on your computer and enter the following command: adb logcat -d > log.txt. This command extracts the log files from your mobile device and saves them in a file named "log.txt" on your computer.

**Reply by saisaigraph** [reply]
Thanks Bob.
Just wondering if it is possible to read these logs directly/indirectly using flet. It looks like it is a no but just checking.
Thanks
Sai
